test/shm_size.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+}
+
+function teardown() {
+  cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+  # Start it with the environment variable set, don't use replace_config
+  CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
+
+  # Run base container to ensure it creates at all
+  run crictl runp <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # Run multi-container pod to ensure that they all work together
+  pod_id="$output"
+  run crictl create "$pod_id" "$TESTDATA"/container_sleep.json <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
+  ctr_id="$output"
+
+  # Start the container by its ID
+  run crictl start "$ctr_id"
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # Confirm that the new size is applied
+  run crictl exec --sync "$ctr_id" df
+  [[ $(echo "$output" | grep '/dev/shm') == *'16384'* ]]
+  echo "$output"
+
+  # Negative tests
```

Comment:
can this case be put in a new test? it'll allow us to have feedback on this case if the positive case fails